install_tools.R

```diff
@@ -0,0 +1,107 @@
+#' @title install_tools.R
+#' @description
+#' This script installs the necessary tools for Automated Evaluation Reports and Projects
+#' @param tools.csv csv file describing the versions of software tools used to generate the reports
+#' @author [Open Systems Pharmacology](https://github.com/Open-Systems-Pharmacology)
+
+#' @description Install CRAN R packages
+install.packages(c("dplyr", "purrr", "covr", "readr", "tidyr", "webshot", "spelling", "readxl", "data.table", "gridtext", "ggtext", "tidyselect", "testthat", "rmarkdown", "rsvg", "svglite", "cowplot"), repos = "http://cran.us.r-project.org", type = "win.binary")
+
+toolsData <- read.csv("tools.csv", stringsAsFactors = FALSE, colClasses = "character")
+print(toolsData)
+
+#' @title installTool
+#' @description
+#' Install tool by tool name and version
+#' TODO: this may be changed later on because newer versions of OSPSuite may rely on `{remotes}` package directly
+#' @param toolName Identifier of the tool to install
+#' @param toolsData data.frame mapping tool identifier and version
+installTool <- function(toolName, toolsData = toolsData) {
+  toolVersion <- toolsData$Version[toolsData$Tool %in% toolName]
+  if (is.na(toolVersion)) {
+    return()
+  }
+  toolPath <- switch(toolName,
+    "OSP Suite" = paste0(

```

Comment:
let's rename this to e.g. **ospsuite-R** (to avoid confusion with the OSP Suite = PK-Sim/MoBi/...)